versions/3.0.4.md

```diff
@@ -2402,7 +2400,7 @@ links:
       username: $response.body#/username
 ```
 
-or an absolute `operationRef`:
+or a URI `operationRef`:
```

Comment:
This correct although the whole thing feels awkward.  But not enough to change it- this is good.